valgrind/docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:24.04
+
+RUN apt-get update && apt-get install -y git
```

Comment:
If possible I would try to get all installs in a single run, unless there is a good reason to have them at different layers throughout the file

Reply:
Mostly keeping them separate to have a very clear list of dependencies listed for each of Postgres, Citus and the other main things.